pkg/apis/openebs/v1alpha1/devicerequest_types.go

```diff
@@ -2,21 +2,51 @@ package v1alpha1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 )
 
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
 // DeviceRequestSpec defines the desired state of DeviceRequest
 type DeviceRequestSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
+	Capacity  uint64 `json:"capacity"`  // disk size in bytes
+	DriveType string `json:"driveType"` // DriveType represents the type of drive like SSD, HDD etc.,
+	HostName  string `json:"hostName"`  // Node name from where device has to be claimed.
 }
 
+type PoolClaimInfo struct {
+	APIVersion   string    `json:"kind,omitempty" protobuf:"bytes,2,opt,name=apiVersion"`
+	Kind         string    `json:"kind,omitempty" protobuf:"bytes,1,opt,name=kind"`
+	Name         string    `json:"name,omitempty" protobuf:"bytes,3,opt,name=name"`
+	PoolClaimUID types.UID `json:"poolClaimUID" protobuf: "bytes,4,opt,name=deviceClaimUID,casttype=k8s.io/apimachinery/pkg/types.UUID"`
+}
+
+// DeviceClaimPhase is a typed string for phase field of DeviceClaim.
+type DeviceClaimPhase string
```

Comment:
Can the Claim be replaced with `Request` whereever applicable.

Reply:
Done.